LR_Tourist/BLL/Model/HotelDTO.cs

```diff
@@ -0,0 +1,21 @@
+﻿
+using DA.Data;
+
+namespace BLL.Model
+{
+    public class HotelDTO: IEntityBase
+    {
+        public int Id { get; set; }
+        public string NameHotel { get; set; }
+
+        public int Star { get; set; }
+
+        public string Phone { get; set; }
+
+        public override string ToString()
```

Comment:
I guess it's a not DTO.
https://en.wikipedia.org/wiki/Data_transfer_object
**Refactor all models in DL and BLL.**

Reply:
I your case - object in DL can be a DTO, but DTO - object used as WebAPI entities.
It means, your project have business objects (for example, this object) and database entities (a kind of DTOs).

Reply:
fixed